### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by nilsleh**

### CODE REVIEW

**Comment by nilsleh** [comment]
@adamjstewart I just copied the file from torchgeo

**Review by adamjstewart — COMMENTED** [review]
Never copy-n-paste files without knowing what they mean. Read the link at the top.

The presence of this file is not enough to run dependabot. If you check the repo settings, there should be something about dependencies or dependabot that allows you to enable dependabot for the repo.

We should consider adding minimum dependency version tests. Remind me to do this next week.

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      torch:
+        patterns:
+          - 'torch'
+          - 'torchvision'
+    ignore:
+      # setuptools releases new versions almost daily
+      - dependency-name: 'setuptools'
+        update-types: ['version-update:semver-patch']
+      # segmentation-models-pytorch pins timm, must update in unison
+      - dependency-name: 'timm'
```
[the remark this answers is not in the document]
Comment (nilsleh) [reply .github/dependabot.yaml:27]: I do use timm

[the remark this answers is not in the document]
Comment (adamjstewart) [reply .github/dependabot.yaml]: Ah, I was only looking at required.txt, my bad

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
FYI, the next smp release will unpin timm, so you'll be able to remove that skip soon.

**Pull request merged**